rust/tw_any_coin/tests/chains/solana/solana_sign.rs

```diff
@@ -549,6 +555,7 @@ fn test_solana_sign_create_and_transfer_token() {
         // 0.0029
         amount: 2900,
         decimals: 6,
+        is_token_2022: false,
         ..Proto::CreateAndTransferToken::default()
```

Comment:
You can ignore `is_token_2022` field here as `..Proto::CreateAndTransferToken::default()` leaves it default (false) value. Same in other places